src/components/WebexAudioSettings/WebexAudioSettings.scss

```diff
@@ -0,0 +1,8 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-audio-settings {
+  h5 {
```

Comment:
please also add the margins, h5 elements have their own margins that we might need to override.

Reply:
and you can keep using h2 if everything is specified